src/bpmncwpverify/bpmn.py

```diff
@@ -290,6 +326,35 @@ def _build_graph(self, process: Process) -> None:
 
                     # update source node's out flows array
                     process[source_ref].add_out_flow(flow)
+                    # update target node's in flows array
+                    process[target_ref].add_in_flow(flow)
+
+    def _traverse_messages(self, root: Element) -> None:
```

Comment:
I assume that `traverse` is the new name for `accept`? Is that correct?

Reply:
@ericmercer No, this is not apart of the visitor pattern. This is part of the bpmn graph creation that walks through the xml to get the interprocess messages. Is this poor naming?